frontend/public/index.html

```diff
@@ -22,6 +22,14 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Kiali</title>
+    <!-- Preload the pficon font to ensure it is available immediately for Graph edges or ace editors -->
+    <link rel="preload" href="%PUBLIC_URL%/pficon.woff2" as="font" type="font/woff2" crossorigin />
```

Comment:
Did you check if this link works without the "cross-origin" tag?

They shouldn't be open as it's a source of high vulnerabilities, a couple of similar links on that page don't use it:
![image](https://user-images.githubusercontent.com/1662329/174002859-b650582d-a4bb-4751-ba9b-e01484cac0cd.png)